src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ModifiedControlVariableCheck.java

```diff
@@ -92,8 +113,15 @@
  * <p>Example:</p>
  *
  * <pre>
- * for (String line: lines) {
- *   line = line.trim();   // it will skip this violation
+ * import java.util.*;
+ * class Demo {
+ *    public static void main(String args[]) {
+ *      List&lt;String&gt; lines =new ArrayList&lt;String&gt;();
+ *      lines.add(" CHECKSTYLE ");
+ *      for (String line: lines) {
```

Comment:
this is overkill, please read
https://www.geeksforgeeks.org/for-each-loop-in-java/

please use 
```
for (String arg: args) {
  arg = arg.trim();   // ok
}
```
no imports are required.
please add this example to code for default config to show that violation will be on that line.

Reply:
@romani  sure I will replace it. But is there any mistake in my example?

Reply:
No mistake, just keep it simple and avoid extra code